src/scss/features/_nav.scss

```diff
@@ -1,7 +1,7 @@
 @mixin oHeaderNav() {
 	.o-header__nav {
-		@include oTypographySans(0);
-		@include oTypographyProgressiveFontFallback('sans', 0);
+		@include oTypographySans($scale: -1, $line-height: 20px);
+		@include oTypographyProgressiveFontFallback('sans', $scale: -1);
```

Comment:
`oTypographySans` includes the progressive fallback by default, so I think we can remove this line?
https://registry.origami.ft.com/components/o-typography@5.7.8/sassdoc#mixin-oTypographySans